src/macroxs_header.F90

```diff
@@ -0,0 +1,728 @@
+module macroxs_header
+
+  use constants,       only: MAX_FILE_LEN, ZERO, ONE, TWO, PI
+  use list_header,     only: ListInt
+  use material_header, only: material
+  use math,            only: calc_pn, calc_rn, expand_harmonic
+  use nuclide_header
+  use scattdata_header
+
+  implicit none
+
+!===============================================================================
+! MACROXS_* contains cached macroscopic cross sections for the material a
+! particle is traveling through
+!===============================================================================
+
+  type, abstract :: MacroXS_Base
```

Comment:
Same goes here- `MacroXS` is better. Our [style guide](http://openmc.readthedocs.org/en/latest/devguide/styleguide.html#derived-types-and-classes) says no underscores.